tox.ini

```diff
@@ -1,7 +1,10 @@
 #
 # Configuration for tox and pytest
 
+# first five ignores are defaults per http://flake8.readthedocs.io/en/latest/config.html#default
 [flake8]
+ignore = E123, E133, E226, E241, E242, W503, E402
+max-line-length = 77
 exclude = dist,docs,*.egg-info,.git,ref,_scratch,.tox
```

Comment:
I added this to my vimrc:
```
let g:syntastic_python_flake8_args = "--config=./tox.ini"
```
so that I catch when I'm going over 77 characters on this project.

There's a lot of 78 or 79 character long lines in the project, so flake8 doesn't actually pass with this config.

What do you think about having the flake8 config in here?